benchmark-bench.js

```diff
@@ -30,47 +31,59 @@ function select (callback) {
     })
 }
 
-inquirer.prompt([
-  {
-    type: 'confirm',
-    name: 'all',
-    message: 'Do you want to run all benchmark tests?',
-    default: false
-  },
-  {
-    type: 'input',
-    name: 'connections',
-    message: 'How many connections do you need?',
-    default: 100,
-    validate (value) {
-      return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+(argv.length === 0
+  ? inquirer.prompt([
+    {
+      type: 'confirm',
+      name: 'all',
+      message: 'Do you want to run all benchmark tests?',
+      default: false
     },
-    filter: Number
-  },
-  {
-    type: 'input',
-    name: 'pipelining',
-    message: 'How many pipelines do you need?',
-    default: 10,
-    validate (value) {
-      return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+    {
+      type: 'input',
+      name: 'connections',
+      message: 'How many connections do you need?',
+      default: 100,
+      validate (value) {
+        return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+      },
+      filter: Number
     },
-    filter: Number
-  },
-  {
-    type: 'input',
-    name: 'duration',
-    message: 'How long should it take?',
-    default: 40,
-    validate (value) {
-      return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+    {
+      type: 'input',
+      name: 'pipelining',
+      message: 'How many pipelines do you need?',
+      default: 10,
+      validate (value) {
+        return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+      },
+      filter: Number
     },
-    filter: Number
-  }
-]).then((opts) => {
-  if (!opts.all) {
-    select(list => bench(opts, list))
-  } else {
-    bench(opts, choices)
-  }
-})
+    {
+      type: 'input',
+      name: 'duration',
+      message: 'How long should it take?',
+      default: 40,
+      validate (value) {
+        return !Number.isNaN(parseFloat(value)) || 'Please enter a number'
+      },
+      filter: Number
+    }
+  ])
+  : (async () => {
```

Comment:
`async` is not needed.

Reply:
Is was that or Promise.resolve. Do you want me to change the inline function to a resolved promise?

Reply:
If async is an option would you mind to rewrite it to simply use `async/await` without ternarys?